### PR TITLE
Create SP19-101010.md

### PR DESCRIPTION
**Pull request by Xiangmingchen**

### CODE REVIEW

File: _projects/SP19-101010.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: 101010
```
Comment (ShanTulshi) [inline _projects/SP19-101010.md]: This line broke the build :joy:
```suggestion
title: "101010"
```

File: _projects/SP19-101010.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+image: https://drive.google.com/file/d/1ViBTIdTskJJas1hImA3Ytet1czVtIOTw/view?usp=sharing
```
Comment (ShanTulshi) [inline _projects/SP19-101010.md]: Please move the image to `/img/projects/` and link it as such here, it needs a direct link for embed and that is something Google Drive does not allow.

Comment (Xiangmingchen) [reply _projects/SP19-101010.md]: Done.

File: _projects/SP19-101010.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: "101010"
+image: https://raw.githubusercontent.com/illinois-cs498vr/illinois-cs498vr.github.io/master/img/projects/101010.png
```
Comment (ShanTulshi) [inline _projects/SP19-101010.md]:
```suggestion
image: /img/projects/101010.png
```

**Pull request merged**